gnmawsgr/tasks.py

```diff
@@ -58,6 +58,8 @@ def makeshape(itemid,uri,tagname='original',agent=None):
     if not re.match(r'^\w+:/',uri):
         uri = 'file:' + uri
 
+    uri = uri.replace(' ','%2520')
```

Comment:
this is a bandaid fix so is OK for the time being, but it may fail on other characters (see http://apidoc.vidispine.com/latest/storage/uri.html).

It really needs to be either replaced with a proper call to a url encoding - e.g., urllib.quote() a couple of times - or ideally replace with calls to the equivalent methods in gnmvidispine where I already put the time in to make it work.